js/formUtility.js

```diff
@@ -47,6 +52,9 @@ $(document).ready(function(){
         },
         minLength: 2,
         select: function(event, ui){ 
+            if(ga){
+                ga('send', 'event', 'LANDING_PAGE', 'company-query-autocomplete-select', ui['item']['label']);
```

Comment:
I would suggest that we send `ui.item.company_id` back,

Notice that `ui.item.label` may be an array, for example, if you search YAHOO